package.json

```diff
@@ -20,10 +20,9 @@
     "test:jest:stories": "testing=storybook TEST_END_FILES=jest-story jest",
     "test:jest:update": "npm run test:jest -- -u",
     "test:jest:coverage": "npm run test:jest -- --coverage",
-    "start": "npm run build:ts-index && npm run build-react-icons && webpack --mode production --watch --config webpack/default.config.js",
-    "build": "npm run build:exports && npm run build-react-icons && webpack --mode production --config webpack/default.config.js",
-    "build:exports": "./scripts/build-exports.js",
-    "build:ts-index": "./scripts/build-ts-index.js",
+    "start": "npm run build-react-icons && webpack --mode production --watch --config webpack/default.config.js",
+    "build": "npm run build-react-icons && webpack --mode production --config webpack/default.config.js && npm run build:esm",
```

Comment:
why moving build:esm to be last? I think we moved it to be first for a reason (not remember why)

Reply:
Where was it? 
Haven't moved in on purpose